### DISCUSSION TITLE
Request: ユーザごとのブックマークを常に表示したい

### DISCUSSION THREAD

**Discussion by uniocto** [growilabs/growi#7703] [Ideas / Requests] [answered]
## Informations
自分のブックマークから各ページにアクセスする時、常にHOMEページを経由する必要がある。　　
そのため、常にブックマークが表示されていて、ブックマークの各ページにアクセスできるようにして欲しい。

**Describe the solution you'd like**
ログインしているユーザのブックマーク表示を、ページツリーやカスタムサイドバーの並びに追加する。

**Describe alternatives you've considered**
個々人のカスタムサイドバーを表示できるようにする。

**Comment by kaori-tokashiki** [comment]
@uniocto
Issueの起票ありがとうございます。
サイドバーにブックマークを表示する施策は既にデザイン検討から走り出しております。

**Comment by yuki-takei** [comment, marked as answer]
@uniocto v6.1 で実装されました
